### PR TITLE
April 24, 2024 - Contributor Spotlight for Mark Waite

### PR DESCRIPTION
**Pull request by kmartens27**
This PR is for the addition of the contributor spotlight page for @MarkEWaite . The intended publication date is April 24th.

It also includes the feature flag change to false for Hervé's spotlight. I also realized the publication date for Hervé was incorrect so this updates that to be aligned with the correct date.

### CODE REVIEW

File: src/pages/contributors/mark-waite.adoc [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+Do not in believe self-imposed barriers to open source contribution.
+I mistakenly thought that a mere mortal, such as myself, could not possibly be of use to a successful and impressive project like Jenkins.
+I was wrong.
+Mere mortals are _exactly_ the type of people who do _amazing_ work in the Jenkins project.
+Find a topic that interests you, either professionally or personally.
+Find areas in the Jenkins project that are involved in that area of interest.
+Contribute your skills and your passion in that area.
+Help others do the same.
+
+I was interested in test automation, so I wrote automated tests for the Jenkins git plugin.
```
Comment (ammanders) [inline src/pages/contributors/mark-waite.adoc:78]:
```suggestion
I was interested in test automation, so I wrote automated tests for the Jenkins Git plugin.
```
We usually initial cap references to Git in our docs. Please change throughout.

**Review by ammanders — APPROVED** [review]

File: src/pages/contributors/mark-waite.adoc (hunk 1 above)
Comment (kmartens27) [reply src/pages/contributors/mark-waite.adoc]: thanks very much @ammanders!

File: src/pages/contributors/mark-waite.adoc [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+I often found that the flexibility of Jenkins was more important to me than the specific reasons that an employer had chosen another solution.
+I had problems to solve and Jenkins solved those problems.
+
+== What problems has Jenkins solved for you?
+
+Jenkins builds my code, runs my tests, and shows the results across a wide range of systems and technologies.
+
+== Is there an aspect of Jenkins that you're particularly passionate about?
+
+I have been involved in each area of Jenkins at some point, but I'm proud of the link:https://plugins.jenkins.io/git/[Git] and link:https://plugins.jenkins.io/git-client/[Git client] plugins that I created and maintain.
```
Comment (MarkEWaite) [inline src/pages/contributors/mark-waite.adoc:59]: I definitely did not create either of those plugins.  The git plugin was created by Nigel Magnay with significant contributions from Kohsuke Kawaguchi, Nicolas de Loof, Robert Sandell, and many others.  The git client plugin was created by Nicolas de Loof with contributions from Kohsuke Kawaguchi and many others.  I have written many of the tests for those two plugins, but was definitely not the creator of either of them.
```suggestion
I have been involved in each area of Jenkins at some point, but I'm proud of the link:https://plugins.jenkins.io/git/[Git] and link:https://plugins.jenkins.io/git-client/[Git client] plugins that I maintain.
```

File: src/pages/contributors/mark-waite.adoc [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+I'm also involved in the platforms and documentation SIGs, where I host the later Jenkins documentation office hours.
```
Comment (MarkEWaite) [inline src/pages/contributors/mark-waite.adoc:60]: Later seems less clear than calling it "Asia".
```suggestion
I'm also involved in the platforms and documentation SIGs, where I host the Jenkins documentation office hours (Asia).
```

**Review by MarkEWaite — CHANGES_REQUESTED** [review]
One change that is required and the other that is optional.

**Review by MarkEWaite — APPROVED** [review]
I merged the changes that I proposed...

File: src/pages/contributors/mark-waite.adoc [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+:page-linkedin: markwaite
+:page-twitter: MarkEWaite
+:page-github: MarkEWaite
+:page-email: 
+:page-image: avatar/mark-waite.jpg
+:page-pronouns: He/Him/His
+:page-location: Loveland, Colorado, USA
+:page-firstcommit: 2008
+:page-datepublished: 2024-04-29
+:page-featured: false
```
Comment (krisstern) [inline src/pages/contributors/mark-waite.adoc:12]:
```suggestion
:page-featured: true
```

**Review by krisstern — APPROVED** [review]
LGTM

**Comment by kmartens27** [comment]
thank you very much for review and edits @MarkEWaite !!

**Pull request merged**